pkg/await/core_service.go

```diff
@@ -253,11 +240,11 @@ func (sia *serviceInitAwaiter) await(
 }
 
 func (sia *serviceInitAwaiter) processServiceEvent(event watch.Event) {
-	inputServiceName := sia.config.currentInputs.GetName()
+	inputServiceName := sia.config.currentOutputs.GetName()
```

Comment:
Another change from inputs to outputs here, this one seems to be harmless. Just wondering if it was on purpose.

Reply:
I'll revert and see. Can't remember why at this point.